<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestion De Parents</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>571</width>
       <height>331</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Ajout_Parent">
      <attribute name="title">
       <string>Ajouter Parent</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>190</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cin">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="mail">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>190</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>230</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Mot de Passe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mdp">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>230</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Affiche_Parent">
      <attribute name="title">
       <string>Afficher Parents</string>
      </attribute>
      <widget class="QTableView" name="tab_parents">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>521</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Supprime_Parents">
      <attribute name="title">
       <string>Supprimer Parents</string>
      </attribute>
      <widget class="QLineEdit" name="Scin">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Supprimer">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Modifier_Parent">
      <attribute name="title">
       <string>Modifier Parent</string>
      </attribute>
      <widget class="QPushButton" name="Modifier">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>270</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Mot de Passe</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="MCin">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="MNom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Mprenom">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="MMail">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="MMdp">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Option Avanc√©es</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>9</y>
       <width>591</width>
       <height>321</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="chercher">
      <attribute name="title">
       <string>Chercher</string>
      </attribute>
      <widget class="QLineEdit" name="R_CIN">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Rechercher">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QTableView" name="Rechercher_Parent">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>511</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tri">
      <attribute name="title">
       <string>Tri</string>
      </attribute>
      <widget class="QPushButton" name="tri_croissant">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri Par Ordre Croissant de CIN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_decroissant">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri Par Ordre Croissant de CIN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tri_ordre_alphabetique">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>191</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri Par Ordre Alphabetique</string>
       </property>
      </widget>
      <widget class="QTableView" name="Tri_Parent">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>511</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Detection De Mouvement</string>
      </attribute>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>alert</string>
       </property>
      </widget>
      <widget class="QLabel" name="state">
       <property name="geometry">
        <rect>
         <x>76</x>
         <y>122</y>
         <width>251</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
